doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1344,6 +1364,27 @@ The fully expanded example above (without environment variables) looks like this
 
   curl -H "X-Dataverse-key: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" -X DELETE "https://demo.dataverse.org/api/datasets/24/versions/:draft"
 
+Deaccession Dataset
+~~~~~~~~~~~~~~~~~~~
+
+Given a version of a dataset, updates its status to deaccessioned.
```

Comment:
As far as I understand, the code allows you to re-deaccession a dataset that has already been deaccessioned, allowing you to update the `deaccessionReason`, for example. Can we mention this here in the guides?

Reply:
Just added a note about this to the docs. As we discussed on Friday, re-deaccessioning is not supported.